pkg/security/probe/probe_windows.go

```diff
@@ -84,6 +85,11 @@ func (p *WindowsProbe) Start() error {
 			select {
 			case <-p.ctx.Done():
 				return
+
+			case <-p.onError:
+				// in this case, we got some sort of error that the underlying
+				// subsystem can't recover from.  Need to initiate some sort of cleanup
```

Comment:
What circumstances could this happen in and is the cleanup coming in a future PR? At a minimum, is it worth logging an error? If subsystem can't recover should we `return` or `continue`?

Reply:
yes, it's coming in a future PR.
Paul & I agreed that the best way to integrate this was to get this code in, so that the security team could have a clean  starting point on main to appropriately handle.
This is intended to be a no-op for now.